framework/i18n/Formatter.php

```diff
@@ -1296,6 +1296,8 @@ public function asCurrency($value, $currency = null, $options = [], $textOptions
      * @return string the formatted result.
      * @throws InvalidArgumentException if the input value is not numeric or the formatting failed.
      * @throws InvalidConfigException when the [PHP intl extension](http://php.net/manual/en/book.intl.php) is not available.
+     *
+     * This formatter does not work well with very big numbers.
      */
```

Comment:
this text will now be part of the `@throws` tag ;)

Reply:
fixed in https://github.com/yiisoft/yii2/commit/fe07177dddeaf4b8d9c1dcbc103ba3d6f572cb10